manuscript/the-meat.txt

```diff
@@ -26,7 +26,7 @@ This will asynchronously load a CSV file, parse it, and return the result in the
 
 Using `setState()` triggers a re-render and should generally be avoided because relying on state can cause inconsistencies in your UI. It’s very appropriate to store 12MB of raw data as state, though.
 
-But the data we’ve got now has keys with spaces like `submit date` and `job title`. And everything is a string. Not very nice to work with.
+So the data we’ve got now has keys with spaces like `submit date` and `job title`, and everything is a string. This is less than ideal, and not very nice to work with.
```

Comment:
We can drop the connecting first word since it doesn't really add anything. I'd also change last sentence to `This is not very nice to work with`.